chandra_aca/drift.py

```diff
@@ -173,3 +174,41 @@ def get_aca_offsets(detector, chip_id, chipx, chipy, time, t_ccd):
     ddz = dz_chip - dz_pred
 
     return ddy, ddz
+
+
+def get_web_zero_offset_table():
+    """
+    Get official SOT MP zero offset aimpoint table:
+
+    :returns: zero offset aimpoint table as astropy.Table
+    """
+    ZOA = "https://icxc.harvard.edu/mp/html/aimpoint_table/zero_offset_aimpoints.txt"
+    page = requests.get(ZOA)
+    return Table.read(page.content, format="ascii")
+
+
+def get_target_aimpoint(date, cycle, detector, too=False, zero_offset_table=None):
+    """
+    Given date, proposal cycle, and detector, return aimpoint chipx, chipy, chip_id
+
+    :param date: observation date
+    :param cycle: proposal cycle of observation
+    :param detector: target detector
+    :param too: boolean. If target is TOO use current cycle not proposal cycle.
+
+    :returns: astropy table row of chipx, chipy, chip_id
+    """
+    if zero_offset_table is None:
+        zero_offset_table = get_web_zero_offset_table()
+    # Get entries for this detector before the 'date' given
+    filtered_table = zero_offset_table[
+        (zero_offset_table['detector'] == detector)
+        & (DateTime(zero_offset_table['date_effective']).secs <=
+           DateTime(date).secs)]
+    # If this is a TOO, return the most recent entry values
+    if too:
+        return filtered_table[['chipx', 'chipy', 'chip_id']][-1]
+    # Otherwise, return the last match earlier than the effective cycle
+    else:
+        return filtered_table[['chipx', 'chipy', 'chip_id']][
```

Comment:
The return value should either be the entire row (perhaps convenient for the calling function for other reasons?) OR a tuple of three scalars.  I would favor the former but don't feel too strongly.  If the former, then the following would do the trick:
```
return zero_offset_table[ok][-1]
```